packages/device_info/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.2+2 - Oct 18, 2017
```

Comment:
This should reflect the version number in the pubspec.yaml. And since it looks like there was no release for the previous change, both bullets should be under the same version.

Not 100% sure what to do with the date (@mit-mit: what was the date intended for?). Maybe just set it to todays date?

Reply:
@goderbauer I merged the two entries and set the date to today.